lnrpc/walletrpc/psbt.go

```diff
@@ -42,29 +43,41 @@ func verifyInputsUnspent(inputs []*wire.TxIn, utxos []*lnwallet.Utxo) error {
 
 // lockInputs requests a lock lease for all inputs specified in a PSBT packet
 // by using the internal, static lock ID of lnd's wallet.
-func lockInputs(w lnwallet.WalletController,
-	outpoints []wire.OutPoint) ([]*base.ListLeasedOutputResult, error) {
+func lockInputs(w lnwallet.WalletController, outpoints []wire.OutPoint,
+	customLockID *wtxmgr.LockID, lockExpirationSeconds uint64) (
+	[]*base.ListLeasedOutputResult, error) {
 
 	locks := make(
 		[]*base.ListLeasedOutputResult, len(outpoints),
 	)
 	for idx := range outpoints {
 		lock := &base.ListLeasedOutputResult{
 			LockedOutput: &wtxmgr.LockedOutput{
-				LockID:   chanfunding.LndInternalLockID,
 				Outpoint: outpoints[idx],
 			},
 		}
 
+		if customLockID != nil {
```

Comment:
nit: leave the default lnd internal lock ID in the definition above and just overwrite it here if a custom one is provided? No need for the `else`. Same for the duration.

Reply:
Done.